### PR TITLE
A better tab bar for more harmonized design

### PR DESCRIPTION
**Pull request by pmkary**
Hi everyone! I love Black Screen for it's awesome intelliSense like feature, but the truth is the very small tab bar didn't fit the design:

screen shot 2016-11-05 at 8 56 14 pm

The solution I thought would be cool was to change the design to __inset__ title bar and make it a bit bigger (also the search bar got rounded):

screen shot 2016-11-05 at 9 03 10 pm

As you see now the sizes are in harmony. And I took care of every sizing:

screen shot 2016-11-05 at 9 02 54 pm

I really hope that you enjoy this design 👻 !

### P.S
I updated the electron version to fix this: #869 

With love!
Pouya

### CODE REVIEW

**Comment by pmkary** [comment]
I think it may take some time to fully perfect the icons of search bar but it is good for now 🍔

**Comment by drew-gross** [comment]
Cool. I like this, but I think @shockone should approve any big changes to the visual style of Black Screen. I beat you to the electron fix so I reverted that change  for you :p

**Comment by pmkary** [comment]
Ha ha cool! Yeah the design is big change I agree on that! I hope you guys enjoy it. Honestly I checked out black screen months ago and then I got to back to it today and now what I see is fantastic. There are a few areas that I hope I can improve. It's going to be a very fun project to contribute 🍷

**Comment by pmkary** [comment]
In addition to the first commit I also fixed the white screen look on load:

a

Now open with the black background color.

**Comment by drew-gross** [comment]
Cool. If you want to make the background color fix a separate PR I can merge it separately. Also, it looks like the linter is complaining a little on this PR.

**Comment by pmkary** [comment]
@drew-gross that would be nice. I accidentally pushed the background change to this branch and then I though well their both on a UI change so not really big deal. But if you can merge it separately it would be really good.

And yeah. I'm getting used to the work flow...

File: package.json [hunk 1]
```diff
@@ -41,7 +41,8 @@
     "react": "15.3.2",
     "react-dom": "15.3.2",
     "tinycolor2": "1.4.1",
-    "uuid": "2.0.3"
+    "uuid": "2.0.3",
+    "yargs-parser": "4.0.2"
```
Comment (shockone) [inline package.json]: This package isn't used, is it?

File: src/views/css/main.ts [hunk 2]
```diff
@@ -64,7 +64,7 @@ const promptGrid = {
     },
 };
 
-const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight}px)`;
+const sessionsHeight = `(100vh - ${titleBarHeight + infoPanelHeight + 5}px)`;
```
Comment (shockone) [inline src/views/css/main.ts]: How did it become +5?

**Comment by shockone** [comment]
Looks great, @pmkary!

File: package.json (hunk 1 above)
Comment (pmkary) [inline package.json]: Heat it is needed. I tried to pack the app and then this was required. I added it and the build fixed. So I though somehow in the dependencies this one is forgotten, let's add this

File: src/views/css/main.ts (hunk 2 above)
Comment (pmkary) [inline src/views/css/main.ts]: I had to comment the reason. I noticed one thing and it was that when I added to the tab bars height, at scroll you could have seen like 3px of overflow of the content into the tab bar. After playing with the whole thing for a little while, this was the holly number that saved the calculations

Comment (pmkary) [inline src/views/css/main.ts]: I have to say that I really wanted to add something more dynamic to it so a font-size change don't effect the result but no combination worked. It seems that 5 is something that needs to be 5

**Comment by vshatskyi** [comment]
@pmkary I plan to close old PRs to keeps things clean. Is it OK with you if I close this one?

**Comment by pmkary** [comment]
It's completely okay. I forked black screen into [Telephone](https://github.com/pmkary/telephone) and I have it all in there.

**Comment by vshatskyi** [comment]
Looks nice! When it's time to implement themes, I might as well steal yours. ;)

**Pull request closed without merging**